api/oauth.go

```diff
@@ -195,6 +196,11 @@ func completeOAuth(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = err
 		return
 	} else {
+		defer func() {
+			ioutil.ReadAll(body)
```

Comment:
Unless the body is read completely first the connection resource won't release right away. It eventually does but this makes sure it's released immediately